### ISSUE TITLE
Error in writeImpl: Text to be written must be a length-one character vector 

### ISSUE THREAD

**Issue by fengfengyuyu** [statnet/statnetWeb#75]
Hi,

I met a problem as follow,

```
> statnetWeb::run_sw()  

Listening on http://127.0.0.1:6677 
Warning in if (!is.na(attribValue)) { :   
Warning: Error in writeImpl: Text to be written must be a length-one character vector   
[No stack trace available]

```
I use Ubuntu16.04.

plus:  I tried at another computer with win10 , the same issue occurred.

Thanks a lot !

**Comment by pete-arnold** [comment]
Same. Windows 8.1. R 3.6.2.

**Comment by martinamorris** [comment]
Hi @fengfengyuyu this has been fixed now.  Sorry for the delay.  Give it a try and let us know if it works for you.

**Comment by martinamorris** [comment]
fixed with statnet/statnetWeb@a8e4a41
